### DISCUSSION TITLE
CapsLock and NumLock status LEDs not working when xremap service is running

### DISCUSSION THREAD

**Discussion by mastermach50** [xremap/xremap#625] [Q&A]
I am using a usb keyboard plugged into my laptop (installed using nix-flakes). Both my laptop and my keyboard have CapsLock and NumLock status LEDs but they do not work when xremap service is running.

I am running xremap as a system service with watch enabled.

**Comment by k0kubun** [comment]
My keyboard has neither a CapsLock key nor the light, so I can't test and help you myself. However, looking at the logs, @kachick seems to have an idea as to why it currently doesn't work, and maybe even whether it's even possible to let xremap support it or not?

**Comment by kachick** [comment]
Has @mastermach50 remapped CapsLock and NumLock with xremap?
From my experience, all software remapping did not completely help me forget CapsLock even while using udev. 

If I could give advice to myself about this problem a year ago, I would have stopped completely removing the original code. It should have been saved for fallback use with another remapping.
